spec/helpers/jsonParseSafe.js

```diff
@@ -0,0 +1,28 @@
+const { describe, it } = exports.lab = require('lab').script();
+
+const { testRunner } = require('../spec-helpers');
+
+describe('JSONParseSafe helper', () => {
+    const context = {
+        jsonString: '{"name": "John"}',
+        string: 'John Doe',
+    };
+    const runTestCases = testRunner({context});
+
+    it('should have the same behavior as the original if helper', done => {
+        runTestCases([
+            {
+                input: '{{#JSONparseSafe string}}{{name}}{{/JSONparseSafe}}',
```

Comment:
I think this case is outside of the original helper

Reply:
Updated tests